libraries/AP_Motors/AP_MotorsHeli_RSC.cpp

```diff
@@ -283,14 +286,23 @@ void AP_MotorsHeli_RSC::output(RotorControlState state)
                 _control_output = constrain_float( _rsc_arot_bailout_pct/100.0f , 0.0f, 0.4f);
             } else {
                 // set rotor control speed to idle speed parameter, this happens instantly and ignores ramping
-                if (_cooldown_time > 0) {
-                    _control_output = get_idle_output() * 1.5f;
-                    _fast_idle_timer += dt;
-                    if (_fast_idle_timer > (float)_cooldown_time) {
-                        _fast_idle_timer = 0.0f;
+                if (_turbine_start && _starting == true ) {
+                    _control_output += 0.001f;
+                    if(_control_output >= 1.0f) {
```

Comment:
nitpick: small white space issue in this new block.. for example after the "if" should be a space.

Reply:
done. thanks!